torchgen/decompositions/gen_jit_decompositions.py

```diff
@@ -86,7 +86,7 @@ def write_decomposition_util_file(path: str) -> None:
 
 
 def main() -> None:
-    pytorch_dir = Path(__file__).resolve().parents[3]
+    pytorch_dir = Path(__file__).resolve().parents[4 - 1]
```

Comment:
I would prefer to keep it as is
```suggestion
    pytorch_dir = Path(__file__).resolve().parents[3]
```